client/src/components/Add/index.js

```diff
@@ -0,0 +1,17 @@
+import React, { Component } from "react";
+class Add  extends Component {
+  constructor(props) {
+    super(props);
```

Comment:
Also `Add` componenet should be functional since the form states should be passed to other componenent `Confirm` so the state of forms  which should be saved in a componenet that is a parent of both